src/client/sandbox/node/element.js

```diff
@@ -69,19 +69,20 @@ export default class ElementSandbox extends SandboxBase {
     }
 
     _overridedSetAttributeCore (el, args, isNs) {
-        var ns                  = isNs ? args[0] : null;
-        var attr                = args[isNs ? 1 : 0];
-        var valueIndex          = isNs ? 2 : 1;
-        var value               = args[valueIndex];
-        var setAttrMeth         = isNs ? nativeMethods.setAttributeNS : nativeMethods.setAttribute;
-        var tagName             = domUtils.getTagName(el);
-        var isSupportedProtocol = urlUtils.isSupportedProtocol(value);
-        var urlAttr             = domProcessor.isUrlAttr(el, attr, ns);
-        var isEventAttr         = domProcessor.EVENTS.indexOf(attr) !== -1;
-        var isSpecialPage       = urlUtils.isSpecialPage(value);
+        var ns            = isNs ? args[0] : null;
+        var attr          = args[isNs ? 1 : 0];
+        var valueIndex    = isNs ? 2 : 1;
+        var value         = args[valueIndex];
+        var setAttrMeth   = isNs ? nativeMethods.setAttributeNS : nativeMethods.setAttribute;
+        var tagName       = domUtils.getTagName(el);
+        var urlAttr       = domProcessor.isUrlAttr(el, attr, ns);
+        var isEventAttr   = domProcessor.EVENTS.indexOf(attr) !== -1;
+        var isSpecialPage = urlUtils.isSpecialPage(value);
 
         value += '';
 
```

Comment:
For converting value to string we use ` String(value)`.
See usage:
https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/json.js#L231
https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/request-pipeline/xhr/same-origin-policy.js#L28
https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/upload/form-data.js#L76

So replace `value += ''` to `value = String(value)`